### PR TITLE
Add ResourceMapper changes to 5.0.0 release notes

### PR DESCRIPTION
**Pull request by rubensworks**
As requested in #951, this explains the steps required to migrate data files (#949) when updating to 5.0.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # History
 
+#### 5.0.0 Upgrade Notes
+
+- Deprecating support for extensionless files. (**Intervention needed when updating from < 5.0.0!**)
```
Comment (RubenVerborgh) [inline CHANGELOG.md]: Extensionless files are still supported; they are just not assumed to be Turtle anymore.

How about "As of v5.0.0, all Turtle files need an extension"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+    - **How to upgrade?**
+        1. Update node-solid-server to 5.0.0
+        2. Make a backup of your `data/` folder
```
Comment (RubenVerborgh) [inline CHANGELOG.md]: 👍

Comment (RubenVerborgh) [reply CHANGELOG.md]: Also config!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
 # History
 
+#### 5.0.0 Upgrade Notes
+
+- Deprecating support for extensionless files. (**Intervention needed when updating from < 5.0.0!**)
+    - **How to upgrade?**
```
Comment (RubenVerborgh) [inline CHANGELOG.md]: Step 0: stop the server

**Comment by rubensworks** [comment]
👍 Updated!

**Pull request merged**